src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -940,6 +940,56 @@ namespace cryptonote
     typedef epee::misc_utils::struct_init<response_t> response;
   };
 
+  struct COMMAND_RPC_GETMINERDATA
+  {
+    struct request_t: public rpc_request_base
```

Comment:
I don't see a corresponding change to ZMQ-RPC. Add in a later patch, or ? Because I think this requires pool implementations to use both interfaces (at least for the initialization).

Reply:
ZMQ-RPC doesn't have half of needed APIs - not even get_block_template/submit_block, so pool implementations are forced to use regular RPC.

Reply:
Seems like something to add in another PR then.